todo_app/app.py

```diff
@@ -1,11 +1,27 @@
-from flask import Flask
+from flask import Flask, render_template
+from flask import request
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+
+from todo_app.flask_config import Config 
 
-from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
```

Comment:
You're allowing POST requests, but then treating them identically to GET requests; it may feel like a positive to allow things to be more flexible but it can easily lead to confusion when users have assumptions. Here, I'd recommend restricting each of these endpoints to only their intended method, e.g. `GET` here and `POST` below